bugout/settings.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 REQUESTS_TIMEOUT = 5
-REQUESTS_TIMEOUT_RAW = os.environ.get("BUGOUT_REQUESTS_TIMEOUT")
+REQUESTS_TIMEOUT_RAW = os.environ.get("BUGOUT_REQUESTS_TIMEOUT_SECONDS")
```

Comment:
Put a default value here: `REQUESTS_TIMEOUT_RAW = os.environ.get("BUGOUT_TIMEOUT_SECONDS", "5")`

Reply:
You can use a value of `-1` to specify no timeout.